### ISSUE TITLE
[Bug]: PDF used in markdown document is not shown content

### ISSUE THREAD

**Issue by mvangriet** [nextcloud/text#4037]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on Github _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] Nextcloud Server **is** running on 64bit capable CPU, PHP and OS.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

In a markdown document a PDF document is inserted. The PDF document is present in the .attachement folder. When I double click on the PDF document a place holder is opened but the content of the PDF document is not shown

### Steps to reproduce

1. Create new markdown document and inset PDF document
2. Close en reopen markdown document
3. Double click on the PDF document, a place holder is shown but the content is missing

### Expected behavior

When I double click on the PDF document I expect that the content of the PDF document is shown and that it is possible to move to the next page of the PDF document

### Installation method

Community Web installer on a VPS or web space

### Nextcloud Server version

26

### Operating system

Debian/Ubuntu

### PHP engine version

PHP 8.0

### Web server

Apache (supported)

### Database engine version

MariaDB

### Is this bug present after an update or on a fresh install?

Updated to a major version (ex. 22.2.3 to 23.0.1)

### Are you using the Nextcloud Server Encryption module?

None

### What user-backends are you using?

- [X] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### List of activated Apps

```shell
This command gives following error on my system;
sudo: unknown user: www-data
sudo: error initializing audit plugin sudoers_audit
```

**Comment by juliushaertl** [comment]
Thanks for reporting, I'll close this as a duplicate of https://github.com/nextcloud/text/issues/3849 with a possible fix in https://github.com/nextcloud/text/pull/3924

**Comment by mvangriet** [comment]
Dear Julius,
Thank you for taking care of my mentioned issue and happy to see that a possible fix is also available. I checked the page more then once but I don’t see what I can do now to solve the issue on my side. Can you once more help me with that?

Groet,
Markus van Griethuijsen
Verzonden vanuit Outlook voor iOS<https://aka.ms/o0ukef>
________________________________
Van: Julius Härtl ***@***.***>
Verzonden: Thursday, April 13, 2023 6:49:55 PM
Aan: nextcloud/text ***@***.***>
CC: mvangriet ***@***.***>; Author ***@***.***>
Onderwerp: Re: [nextcloud/text] [Bug]: PDF used in markdown document is not shown content (Issue #4037)


Closed #4037<https://github.com/nextcloud/text/issues/4037> as not planned.
